assets/content/cookbook/Beginner/Haxe-To-Emscripten-Hello-World.md

```diff
@@ -0,0 +1,65 @@
+[tags]: / "Haxe, Emscripten"
+# Haxe to Emscripten
+Simple hello world  Haxe code. Then compile and linked to emscrpten.
```

Comment:
I would change the intro section to :

> This is a hello world example which will be compiled and linked to [Emscripten](https://emscripten.org/).

You can leave out the thank-you message. It's very friendly but this is not the correct place to write it ❤️ 👍